pkg/harvester/edit/network.harvesterhci.io.clusternetwork/index.vue

```diff
@@ -1,19 +1,62 @@
 <script>
-import Setting from '../harvesterhci.io.setting';
+import CruResource from '@shell/components/CruResource';
+import NameNsDescription from '@shell/components/form/NameNsDescription';
+import ResourceTabs from '@shell/components/form/ResourceTabs/index';
+import Tab from '@shell/components/Tabbed/Tab';
+import Labels from '@shell/components/form/Labels';
+
+import CreateEditView from '@shell/mixins/create-edit-view';
 
 export default {
-  name:       'EditClusterNetwork',
-  components: { Setting },
+  name: 'EditClusterNetwork',
+
+  components: {
+    CruResource,
+    NameNsDescription,
+    ResourceTabs,
+    Tab,
+    Labels,
+  },
+
+  mixins: [CreateEditView],
 
-  props: {
-    value: {
-      type:     Object,
-      required: true,
+  computed: {
+    doneLocationOverride() {
+      return this.value.doneOverride;
     }
   },
 };
 </script>
 
 <template>
-  <Setting :value="value" />
+  <CruResource
+    :resource="value"
+    :mode="mode"
+    :errors="errors"
+    @finish="save"
+  >
+    <NameNsDescription
```

Comment:
It looks like name needs a validator to limit it to 12 characters: when I tried to create a network config I got this error 
![Screen Shot 2022-09-28 at 3 02 21 PM](https://user-images.githubusercontent.com/42977925/192896950-c43ad5eb-f792-40f4-a432-42635267793b.png)

Reply:
Done